ci-operator/jobs/openshift-psap/ci-artifacts/openshift-psap-ci-artifacts-master-presubmits.yaml

```diff
@@ -344,6 +344,133 @@ presubmits:
         secret:
           secretName: result-aggregator
     trigger: (?m)^/test( | .* )4.8-images,?($|\s.*)
+  - agent: kubernetes
+    always_run: true
```

Comment:
optional, always_run

Reply:
fixed for `always_run false`, what's `optional: true`?

Reply:
It means that even when you run the job manually, and it fails, it still doesn't block a merge